examples/Microsoft.DBforPostgreSQL_flexibleServers_configurations@2022-12-01/main.tf

```diff
@@ -0,0 +1,96 @@
+terraform {
+  required_providers {
+    azapi = {
+      source = "Azure/azapi"
+    }
+    azurerm = {
```

Comment:
Please remove the azurerm definition and its provider block if it's not used.

Reply:
Updated